svm/src/account_loader.rs

```diff
@@ -404,6 +405,7 @@ fn get_requested_loaded_accounts_data_size_limit(
     )
 }
 
+#[allow(dead_code)]
 fn account_shared_data_from_program(loaded_program: &ProgramCacheEntry) -> AccountSharedData {
```

Comment:
Should we delete this function, since it's no longer used?

Reply:
We want to bring back the optimization, but its in the GIT history anyway so I am fine with removing it for now.